make/prepare

```diff
@@ -354,16 +357,16 @@ render(os.path.join(templates_dir, "ui", "env"),
         )
 
 registry_config_file = "config_ha.yml" if args.ha_mode else "config.yml"
-storage_provider = rcp.get("configuration", "registry_storage_provider").strip()
-storage_provider_cfg = rcp.get("configuration", "registry_storage_provider_cfg").strip()
-if storage_provider == 'filesystem':
-    if not storage_provider_cfg:
-        storage_provider_cfg = 'rootdirectory: /storage'
-    else:
-        storage_provider_cfg = 'rootdirectory: /storage' + ',' + storage_provider_cfg
+storage_provider_name = rcp.get("configuration", "registry_storage_provider_name").strip()
+storage_provider_config = rcp.get("configuration", "registry_storage_provider_config").strip()
+if storage_provider_name == "filesystem":
+    if not storage_provider_config:
+        storage_provider_config = "rootdirectory: /storage"
+    elif "rootdirectory:" not in storage_provider_config:
+        storage_provider_config = "rootdirectory: /storage" + "," + storage_provider_config
 # generate storage configuration section in yaml format
 storage_provider_info = ('\n' + ' ' * 4).join(
```

Comment:
It should be 8 space right?

Reply:
It's 4 spaces since #3908 Fix indentation in registry yaml files was merged.